content/applications/finance/accounting/bank/reconciliation/use_cases.rst

```diff
@@ -1,105 +1,92 @@
-this invoice as paid immediately in order to indicate that we can start rendering services to our
-customer.
+.. note::
+   - if you wish to bypass the **"In Payment"** status, click :ref:`here <bypass>`
 
-Our customer send us a payment confirmation. We can thus register a
-payment and mark the invoice as paid.
+Bank reconciliation
+===================
 
-.. image:: use_cases/use02.png
-   :align: center
+Payment registration
+--------------------
 
-By clicking on **register payment,** we are telling Odoo that our
-customer has paid the Invoice. We thus have to specify the amount and the
-payment method.
+If you have received a payment confirmation from your customer, or if you have made a payment to one
+of your vendors, click on :guilabel:`Register Payment` from within the related invoice or bill. Once
+you click, a pop-up window will appear prompting you to input the payment method and other related
+payment information. When done, click :guilabel:`Create Payment`. The payment status of the document
+will change to "In Payment".
 
-.. image:: use_cases/use03.png
+.. image:: use_cases/register-payment.png
    :align: center
+   :alt: The Register Payment button.
 
-Now we can always find the payment details in the Invoice by clicking on the
-:menuselection:`Info --> Open Payment`.
-
-.. image:: use_cases/use04.png
+.. image:: use_cases/pop-up-window.png
    :align: center
+   :alt: Pop-up window.
 
-The invoice has been paid and **the reconciliation has been done
-automatically.**
+Bank statement matching
+-----------------------
 
-Case 2: Bank statements reconciliations
----------------------------------------
+When the payment is registered, you must now match it with its corresponding bank statement. Head to
+the **Accounting Dashboard**, click on the **Bank** journal, and create a new bank statement. If you
+have synchronized your Odoo instance with your bank account, select the imported bank statement that
+matches your invoice or bill.
 
-We start at our issued Invoice of 3000 euros for Smith & Co. Let's also
-assume that other Invoices are open for different customers.
-
-.. image:: use_cases/use05.png
+.. image:: use_cases/bank-statements.png
    :align: center
+   :alt: Bank statements menu.
 
-We receive our bank statement and find that not only the invoice issued to Smith & Co has
-been paid, but the one to Buzz of 92 euros as well.
+Once you have populated the required fields, click on **Post** and then **Reconcile**. You will land
+on the reconciliation page, where you can match the payment with the corresponding bank statement.
+Odoo automatically suggests matches based on the **Label**, **Partner**, and **Amount** entered upon
+the creation or import of the bank statement. However, if no matches are suggested or if the match
+does not correspond with its counterpart entry, you can always change the partner, or select another
+payment from the list by clicking on the blue line (cf. screenshot below).
```

Comment:
```suggestion
payment from the list by clicking on the blue line (see the screenshot below).
```
_Style guides recommend that cf. be used only to suggest a comparison, and the word 'see' be used to point to a source of information._(https://en.wikipedia.org/wiki/Cf.#cite_note-2)